<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_connection">
    <attribute name="title">
     <string>Connection</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="title">
      <string>Baudrate</string>
     </property>
     <widget class="QComboBox" name="combo_baud">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <property name="title">
      <string>COM port</string>
     </property>
     <widget class="QComboBox" name="combo_com">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>231</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Network</string>
     </property>
     <widget class="QLineEdit" name="ip">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mask:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mask">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Instruments</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>301</width>
       <height>50</height>
      </rect>
     </property>
     <property name="title">
      <string>Analyzer</string>
     </property>
     <widget class="QLineEdit" name="addr_sa">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>140</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_sa">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>301</width>
       <height>50</height>
      </rect>
     </property>
     <property name="title">
      <string>Generator</string>
     </property>
     <widget class="QComboBox" name="combo_gen">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="addr_gen">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>140</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="title">
      <string>Attenuators (dB)</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>GEN:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SA:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="atten_gen">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="atten_sa">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="title">
      <string>Calibration (MHz)</string>
     </property>
     <widget class="QLineEdit" name="cal_stop">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Start:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cal_start">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Stop:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cal_steep">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Steep:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tests settings</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Versions</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SW:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Patch:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sw_version">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="patch_version">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Authentication</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Login:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="user_name">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="user_pass">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Storm Interface path</string>
     </property>
     <widget class="QLineEdit" name="storm_path">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="storm_path_btn">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>140</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>GPS/GPRS</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>APN:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="apn_name">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>timout:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="modem_timeout">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_utils">
    <attribute name="title">
     <string>Utils</string>
    </attribute>
    <widget class="QPushButton" name="export_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
    <widget class="QPushButton" name="import_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
